src/blog/2024/10/announcement-mqtt-broker.md

```diff
@@ -21,7 +21,7 @@ The MQTT Service is available now on FlowFuse Cloud. FlowFuse permits you to set
 You can now use FlowFuse to manage your own MQTT Clients alongside your Node-RED instances, making it easier to build full-stack, event-driven applications within FlowFuse.
 
 ## Use Cases
-
+FlowFuse MQTT Service facilitates real-time communication, an essential element for optimizing industrial processes. This service allows for the monitoring of production lines and the precise control of processes, leading to enhanced efficiency. Its seamless integration with existing industrial systems simplifies data collection, which can then be analyzed to improve decision-making and boost productivity. Here are some typical uses cases:
```

Comment:
Not sure why this section was added. TBH, IMHO, it feels dangerously close to bordering on "waffle".

Yes, MQTT (like almost any technology) can be used to "improve production" but it is not a magic bullet.

Reply:
It's to give the use cases some context. this is meant to be promotional post ;)

Reply:
Hi Hasmin, sure, i get it, but when I read that section (as a tech person who has done all of the use-cases in real-world scenarios using tech like Direct DB, OPC, SCADA etc) it just had a "smell" - one i suspect others would detect too.

For example: Just because a facility gets MQTT capability, you do not suddenly end up with "the monitoring of production lines" or "precise control of processes". Nether is it seamless (MQTT is not a discoverable protocol).

I think a little less fluff will come across better.

Just my 2C

Reply:
```suggestion
FlowFuse MQTT Service simplifies access to real-time data, an important element in optimizing industrial processes. With the power of FlowFuse, Node-RED and the MQTT service, your integrations are now even easier. Here are some typical uses cases:
```


1. Changed the focus from  "real-time communication" to "real-time data" as the value is in the data not the act of communicating.
1. I dont spell out "simplifies data collection" because in reality, you still need somewhere to put it (we dont currently offer a DB)
1. I have removed "precise control of processes" (MQTT is rarely used for process control without safety controllers and such) - I felt it was better to leave that out
1. I have simplified the overall statement because much of the improvements are implicit within the use casees (e.g. OEE stands for Overall Equipment Effectiveness, Predictive Maintenance leads to savings, condition Based monitoring improves MBD time, etc)


WDYT? Happy to iterate.